src/Hooks/LandingPages.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace WPML\PB\Elementor\Hooks;
+
+use WPML\FP\Obj;
+
+class LandingPages implements \IWPML_Frontend_Action, \IWPML_Backend_Action, \IWPML_DIC_Action {
+
+	const POST_TYPE = 'e-landing-page';
+
+	/** @var \SitePress $sitepress */
+	private $sitepress;
+
+	public function __construct( \SitePress $sitepress ) {
+		$this->sitepress = $sitepress;
+	}
+
+	public function add_hooks() {
+		add_filter( 'post_type_link', [ $this, 'adjustLink' ], PHP_INT_MAX, 2 );
+	}
+
+	/**
+	 * @see \Elementor\Modules\LandingPages\Module::remove_post_type_slug
+	 *
+	 * @param string   $postUrl
+	 * @param \WP_Post $post
+	 *
+	 * @return string
+	 */
+	public function adjustLink( $postUrl, $post ) {
+		if ( self::POST_TYPE === $post->post_type ) {
```

Comment:
@strategio Wouldn't it better to use `get_post_type()`?

Reply:
Why would it be better @dgwatkins?

```php
function get_post_type( $post = null ) {
	$post = get_post( $post );
	if ( $post ) {
		return $post->post_type;
	}

	return false;
}
```

Reply:
Because we use the WP function designed for this purpose.

If you think its fine, just ignore me. I think of it as forward compatibility :-)